relay-server/src/utils/dynamic_sampling.rs

```diff
@@ -101,6 +101,51 @@ pub fn get_sampling_result(
     SamplingResult::determine_from_sampling_match(sampling_result)
 }
 
+pub fn is_trace_fully_sampled(
+    processing_enabled: bool,
```

Comment:
For a follow-up PR: The `processing_enabled` flag doesn't really belong here, and if we trace it in, then it's just to emit an error warning. Could we look at an alternative way or log this unconditionally?

Reply:
We would have to really try to lift that out and see how we can implement sampling either without or with a semantically more meaningful value.